### ISSUE TITLE
`is_multilabel` is not a valid parameter

### ISSUE THREAD

**Issue by secsilm** [google-ai-edge/mediapipe#6209]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Ubuntu 22.04

### Python Version

3.10

### MediaPipe Model Maker version

0.2.1.4

### Task name (e.g. Image classification, Gesture recognition etc.)

text classification

### Describe the actual behavior

I want to perform multi-label text classification. I noticed the [code](https://github.com/google-ai-edge/mediapipe/blob/12cf718f88ba3b32d0293d2f832b18f3378af747/mediapipe/model_maker/python/text/text_classifier/text_classifier.py#L398) includes the is_multilabel parameter, but after installation, I found that this parameter is not available.

### Describe the expected behaviour

This parameter should be included.

### Standalone code/steps you may have used to try to get what you need

1. `pip install mediapipe-model-maker`; 2. `from mediapipe_model_maker import text_classifier`; 3. `text_classifier.BertHParams?`

### Other info / Complete Logs

```shell
from mediapipe_model_maker import text_classifier


text_classifier.BertHParams(
    learning_rate: float = 3e-05,
    batch_size: int = 48,
    epochs: int = 2,
    steps_per_epoch: Optional[int] = None,
    class_weights: Optional[Mapping[int, float]] = None,
    shuffle: bool = False,
    repeat: bool = False,
    export_dir: str = '/tmp/tmpg422ctjg',
    distribution_strategy: str = 'off',
    num_gpus: int = 0,
    tpu: str = '',
    end_learning_rate: float = 0.0,
    optimizer: mediapipe_model_maker.python.text.text_classifier.hyperparameters.BertOptimizer = <BertOptimizer.ADAMW: 'adamw'>,
    weight_decay: float = 0.01,
    desired_precisions: Sequence[float] = <factory>,
    desired_recalls: Sequence[float] = <factory>,
    gamma: float = 2.0,
    tokenizer: mediapipe_model_maker.python.text.text_classifier.bert_tokenizer.SupportedBertTokenizers = <SupportedBertTokenizers.FULL_TOKENIZER: 'fulltokenizer'>,
    checkpoint_frequency: int = 0,
) -> None
```

**Comment by HarishPermal** [comment]
Hi @secsilm,

Sorry for the delayed response, I've reproduced this with Ubuntu 22.04, python 3.10, and mediapipe-model-maker==0.2.1.4.

The published PyPI wheel does not include the multilabel text-classification implementation although it exists in this [commit](https://github.com/google-ai-edge/mediapipe/commit/3494303a843dd68491e87c7b443b17b267fa866b)

So I tried by installing 0.2.1.4 and overlaying the affected runtime modules from this [commit](https://github.com/google-ai-edge/mediapipe/commit/3494303a843dd68491e87c7b443b17b267fa866b) and it worked, you can give it a try

And MediaPipe Model Maker is officially [deprecated](https://ai.google.dev/edge/mediapipe/solutions/model_maker) and no longer actively maintained,  so this should be treated as a workaround rather than a supported long-term solution

Thanks 😊
